lib/super_digit.rb

```diff
@@ -1,15 +1,38 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
-   
+   if n < 10
+    return n
+   end
+   n_digits = n.digits
+   return super_digit(n_digits.sum)
 end
+
+
   
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(k*n)
+# Space Complexity - O(k*n)
 def refined_super_digit(n, k)
```

Comment:
Because you do `concat_dig` which runs `k` times and then `super_digit` which could be written to run `n` times, this would be O(k + n).